OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -278,7 +279,9 @@ public override void Draw()
 			if (renderShroud != null)
 			{
 				foreach (var cell in dirtyShroudCells)
-					UpdateShroudCell(cell);
+					if (world.Map.Contains(cell))
```

Comment:
Cells are (or should be) checked before they are added to dirtyShroudCells, so this is also unnecessary.